doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+* ``method``: A string containing the name of the method to be invoked. Must be “call”. **REQUIRED**
+* ``params``:
+    * ``account_token``: token of the account from which credits will be taken. Each successful call costs one token. **REQUIRED**
+    * ``version``: The version will determine the format of your requests and the format of the server response. Some results can be unavailable in older versions. For the current version 1.2.0, send ‘120’. If not specified, the latest version will be used. **OPTIONAL**.
+    * ``documents``: The invoice must be provided as a string in the ASCII format. The list should contain only one string. If multiple strings are provided only the first string corresponding to a pdf will be processed. If no pdf is found, the first string will be processed. This field is a list only for legacy reasons. The supported extensions are *pdf*, *png*, *jpg* and *bmp*. **REQUIRED**
+    * ``user_infos``: 
+        Information concerning the person to whom the invoice is intended. This informations is not required in order for the service to work but it greatly improves the performance. **REQUIRED** (can be empty).
+        
+        - ``user_company_vat``: VAT number of the client. **OPTIONAL**
+        - ``user_company_name``: Name of the client’s company. **OPTIONAL**
+        - ``user_company_country_code``: Country code of the client. Format: `ISO3166 alpha-2 <https://www.iban.com/country-codes>`_. **OPTIONAL**
+        - ``user_lang``: client language. Format: *language_code + _ + locale* (ex: fr_FR, en_US). **OPTIONAL**
+        - ``user_email``: client email. **OPTIONAL**
+* ``id``: An identifier established by the client. It allows the client to keep track of which response goes with which request. This makes asynchronous calls easier. **REQUIRED**
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+            {
+                "jsonrpc": string,
+                "method": string,
+                "params": {
+                        "account_token": string (hex),
+                        "version": int,
+                        "documents": [string],
+                        "user_infos": {
+                                "user_company_vat": string,
+                                "user_company_name": string,
+                                "user_company_country_code": string,
+                                "user_lang": string,
+                                "user_email": string,
+                        },
+                },
+                "id": string (hex),
+            }
+
+Response
+^^^^^^^^
+
+* ``jsonrpc``: A string specifying the version of the JSON-RPC protocol. It will be “2.0”.
+* ``id``: The identifier you set in the request body.
+* ``result``: 
+    * ``status_code``: code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
```

Comment:
```suggestion
    * ``status_code``: The code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
```